atd-vzd/migrations/default/1709836840391_data_model_init/up.sql

```diff
@@ -0,0 +1,293 @@
+-- we'll do all our work in a schema called `db`
+create schema db;
+
+-- and we'll set the global search path so we don't have to use the `db` prefix during testing
+alter role visionzero set search_path to db, public;
+
+---------------------------------------------
+--- Create tables ---------------------------
+---------------------------------------------
+create table db.road_types (
+    id serial primary key,
+    description text
+);
+
+create table db.cris_crashes (
+    crash_id integer primary key,
+    primary_address text,
+    latitude numeric,
+    longitude numeric,
+    road_type_id integer references db.road_types (id)
+    references db.road_types (id)
```

Comment:
was this extra `references` an accident?

Reply:
correct—thanks. i'm surprised this works 🤔